website/blog/2020-12-19-lithium.md

```diff
@@ -0,0 +1,211 @@
+---
+author: Tomasz Godzik
+title: Metals v0.9.8 - Lithium
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We're happy to announce the release of Metals v0.9.8, which brings along a
+number of smaller improvements, new Scala 3 milestone support and a hot new
+feature.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">99</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">40</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">4</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">5</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">1</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/31?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text, Atom and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- Add in ability to export worksheet evaluations.
+- Scala 3.0.0-M2 support.
```

Comment:
M3?

Reply:
Good catch! I hope that's the last M because I am apparently getting lost here.